### ISSUE TITLE
WGS to cytosure from balsamic? 

### ISSUE THREAD

**Issue by heronikdin** [Clinical-Genomics/BALSAMIC#605]
Hi! 

As the title says, is it possible to upload WGS samples that are run through balsamic onto cytosure? I have seen that there is this filedownload on the rare disease site for CGH files. 
![image](https://user-images.githubusercontent.com/62098218/113828281-818e5c80-9784-11eb-9886-985e351cdf51.png)

Hälsningar Hero

**Comment by hassanfa** [comment]
I think this is more of a BALSAMIC issue to generate a CGH file first.  Please transfer it Daniel, or can I also do that?

**Comment by hassanfa** [comment]
Aah you already did! 👏🏽

**Comment by heronikdin** [comment]
Thank you for the fast reply! The only addtional thing we would like to "see" maybe is somewhat VAF on cytoure, but I am not sure how feasible it is? Other than that, it should not differ more than from the rare disease site. :)

**Comment by dnil** [comment]
Right, that would take a little fiddling with vcf2cytosure and e.g. TIDDIT, but likely doable. When we discussed that for RD, it got stuck on having good examples. If you have example CGH files with e.g. a B-allele track in it from an SNP array, I would be interested in it for reference!

**Comment by heronikdin** [comment]
sorry, dont have any of that...

**Comment by dnil** [comment]
Ping @khurrammaqbool and @ashwini06; we've wrapped the CNVkit and hg38 updates to vcf2cytosure in a new release: https://github.com/NBISweden/vcf2cytosure/releases/tag/0.7.1.
